### ISSUE TITLE
TypeError: hpd() got an unexpected keyword argument 'alpha'

### ISSUE THREAD

**Issue by jkquijas** [treszkai/best#1]
Im using the latest BEST running on Python 3.7.4.

When I try run  `best_out = best.analyze_two(group1_data, group2_data)`, I get the following traceback:

---------------------------------------------------------------------------
```python
TypeError                                 Traceback (most recent call last)
 in 
     12 
     13     best_out = best.analyze_two(group1_data, group2_data)
---> 14     fig = best.plot_all(best_out)
     15     fig.savefig(f'{blackbox}_best_plots.pdf')

~/dev/ascend/lab-client/venv/lib/python3.7/site-packages/best/plot.py in plot_all(best_results, *args, **kwargs)
    555         return plot_all_one(best_results, *args, **kwargs)
    556     elif isinstance(best_results, BestResultsTwo):
--> 557         return plot_all_two(best_results, *args, **kwargs)
    558     else:
    559         raise ValueError('best_results argument is of unknown type')

~/dev/ascend/lab-client/venv/lib/python3.7/site-packages/best/plot.py in plot_all_two(best_results, bins, group1_name, group2_name)
    367     posterior_std2 = trace['Group 2 SD']
    368 
--> 369     std1_min, std1_max = best_results.hdi('Group 1 SD', DISPLAYED_MASS)
    370     std2_min, std2_max = best_results.hdi('Group 2 SD', DISPLAYED_MASS)
    371     std_min = min(std1_min, std2_min)

~/dev/ascend/lab-client/venv/lib/python3.7/site-packages/best/model.py in hdi(self, var_name, credible_mass)
    290         check_credible_mass(credible_mass)
    291 
--> 292         return tuple(pm.hpd(self.trace[var_name], alpha=(1 - credible_mass)))
    293 
    294     def posterior_prob(self, var_name: str, low: float = -np.inf, high: float = np.inf):

~/dev/ascend/lab-client/venv/lib/python3.7/site-packages/pymc3/stats/__init__.py in wrapped(*args, **kwargs)
     22                 )
     23                 kwargs[new] = kwargs.pop(old)
---> 24             return func(*args, **kwargs)
     25 
     26     return wrapped

TypeError: hpd() got an unexpected keyword argument 'alpha'
```

**Comment by jkquijas** [comment]
I found the cause. BEST breaks when PyMC3 version is greater than 3.7. This should be a hard requirement on the requirements.txt

`pymc3==3.7`

**Comment by ThomasHoppe** [comment]
A work around is 

import pymc3 as pm
pm.summary(best_out.trace)

**Comment by treszkai** [comment]
Fixed in version 2.0.2. Thanks for the report.
